.github/workflows/ci_testing.yml

```diff
@@ -73,8 +73,10 @@ jobs:
 
     - name: Tests
       run: |
-        coverage run --source lightning_transformers -m py.test lightning_transformers tests -v --junitxml=junit/test-results-${{ runner.os }}-${{ matrix.python-version }}-${{ matrix.requires }}.xml
-        coverage xml
+        # suggested by https://github.com/Lightning-AI/lightning-transformers/pull/286#issuecomment-1243954206
```

Comment:
Looks like tests passed succesfully without `coverage` (obviously failed later when trying to upload it)

So the issue is probably caused by a bug in coverage.
We could try updating the test dependencies: https://github.com/Lightning-AI/lightning-transformers/blob/master/requirements/test.txt to match those in PL https://github.com/Lightning-AI/lightning/blob/master/requirements/pytorch/test.txt